homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -37,12 +37,7 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self) -> None:
-        """Initialize flow."""
-
-        self._webfsapi_url: str | None = None
-        self._name: str | None = None
-        self._unique_id: str | None = None
+    _webfsapi_url: str | None = None
```

Comment:
This can be set as not-none, and you can revert the "assert" in the code.

```suggestion
    _webfsapi_url: str
```